sdks/java/extensions/sql/datacatalog/build.gradle

```diff
@@ -35,6 +45,8 @@ dependencies {
 
   testCompile project(":sdks:java:extensions:sql:zetasql")
   testRuntimeOnly library.java.slf4j_simple
+  permitUnusedDeclared "org.threeten:threetenbp:1.4.5"
```

Comment:
Nit: please put the `permitUnusedDeclared` lines right under where they are declared, so it's easier to read. e.g. 

```
compile "org.threeten:threetenbp:1.4.5"
permitUnusedDeclared "org.threeten:threetenbp:1.4.5"
```